app-cdr/cdrtools/cdrtools-3.01~a30.recipe

```diff
@@ -0,0 +1,160 @@
+SUMMARY="CD/DVD/Blu-ray premastering and recording software"
+DESCRIPTION="cdrtools is a set of command line programs suitable for creating \
+and recording file system images to CD, DVD and Blu-ray media.
+
+The suite includes the following programs:
+- cdrecord  A CD/DVD/BD recording program 
+- readcd    A program to read CD/DVD/BD media with CD-clone features 
+- cdda2wav  The most evolved CD-audio extraction program with paranoia support 
+- mkisofs   A program to create hybrid ISO9660/JOLIET/HFS filesystems with \
+optional Rock Ridge attributes 
+- isodebug  A program to print mkisofs debug information from media 
+- isodump   A program to dump ISO-9660 media 
+- isoinfo   A program to analyse/verify ISO/9660/Joliet/Rock-Ridge Filesystems 
+- isovfy    A program to verify the ISO-9660 structures 
+- rscsi     A Remote SCSI enabling daemon
+"
+HOMEPAGE="http://cdrecord.org/" 
+COPYRIGHT="
+	1995-2015 Joerg Schilling (cdrecord, readcd)
+	1993-2004 Heiko Eissfeldt, 2004-2015 Joerg Schilling (cdda2wav)
+	1993-1997 Eric Youngdale, 1997-2002 James Pearson, 1997-2015 Joerg Schilling (mkisofs)
+	"
+LICENSE="
+	GNU GPL v2
+	CDDL v1
+	"
+SOURCE_URI="http://downloads.sourceforge.net/project/cdrtools/alpha/cdrtools-3.01a30.tar.bz2"
+CHECKSUM_SHA256="5b9a2f98771c9d0097a1e7640727655ece2864eea95f38e5611af2b2f6e6d9cd"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:gcc
+	cmd:grep
+	cmd:ld
+	cmd:make
+	cmd:sed
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/cdrecord keep-old
+	settings/rscsi keep-old
+	"
+
+SOURCE_DIR="cdrtools-3.01"
+
+PATCHES="cdrtools-3.01~a30.patch"
```

Comment:
Remove newline before PATCHES & SOURCE_DIR, and move them both to just below CHECKSUM.